client/payment-details/payment-method/index.js

```diff
@@ -20,6 +20,7 @@ import IdealDetails from './ideal';
 import P24Details from './p24';
 import SepaDetails from './sepa';
 import SofortDetails from './sofort';
+import KlarnaDetails from 'wcpay/payment-details/payment-method/klarna';
```

Comment:
Why path is so different here? Is it possible to make it consistent with other details components?

Reply:
My bad, IDE...